contxt/models/changetrackingrecord.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+import json
+
+
+class DBOperation(Enum):
+    INSERT = 0
+    UPDATE = 1 
+    UPSERT = 2 
+    DELETE = 3
+
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_ 
+
+class CTRHeader():
+    def __init__(self, forwardable = True, deduplicatable = True):
+        self.forwardable = forwardable
+        self.deduplicatable = deduplicatable
+
+class ChangeTrackingRecord():
+    '''A JSON Serializable ndustrial.io ChangeTrackingRecord(tm) that is
+       ingestable by the ndustrial ETL java/kotlin components.
+       
+       topic: Str           - the name of the canonical being ETL-ed
+       version: Str         - some long identification I'm not sure
```

Comment:
This is the change table version or CDC version number (cursor position)

Reply:
thanks 👍